flux_led/scanner.py

```diff
@@ -174,6 +175,8 @@ class BulbScanner:
     REBOOT_MESSAGE = b"AT+Z\r"
     ALL_MESSAGES = {DISCOVER_MESSAGE, VERSION_MESSAGE, REMOTE_ACCESS_MESSAGE}
     BROADCAST_ADDRESS = "<broadcast>"
+    if platform.system() == "Windows":
+        BROADCAST_ADDRESS = "255.255.255.255"
```

Comment:
I think you could change it for all oses and 255.255.255.255 should work everywhere